src/pages/Application/Confirm.tsx

```diff
@@ -16,68 +16,11 @@ import WithToasterContainer from '../../shared/HOC/withToaster';
 import theme from '../../shared/Styles/theme';
 import * as DashboardText from '../../features/Dashboard/DashboardText';
 
-interface IConfirmAttendanceState {
-  submitted: boolean;
-  submissionBtn: number;
-}
+const ConfirmAttendanceContainer: React.FC = () => {
+  const [submissionBtn, setSubmissionBtn] = useState<number>(0);
```

Comment:
nit: can we change this to `submitButton`? 

I think it makes more sense with how we refer to it internally and considering we use `submitted` too. I'd suggest `submitBtn` but I think that's what we named the component?

Reply:
Reading through this page's code, `submitBtn` purpose isn't very clear (why is it an number?). 0 seems to correspond to a decline button click while 1 is a confirm button click. I'll rename it to `submitButton` for the purpose of this PR, but create an issue to clear up/better document the submission handler for this form.